packages/analytics/addon/components/ca-field-selector-list.hbs

```diff
@@ -1,8 +1,4 @@
-<table
-  class="uk-table uk-table-divider uk-table-hover"
-  uk-overflow-auto
-  {{did-insert this.setupUIkit}}
->
+<table class="uk-table uk-table-divider uk-table-hover" uk-overflow-auto>
```

Comment:
I think `uk-overflow-auto` is a class https://getuikit.com/docs/utility#overflow

Reply:
Yes, changed it.
The tests are also breaking on main... something changed in the floating dependencies